jitbuilder/control/Jit.cpp

```diff
@@ -179,7 +179,7 @@ extern "C"
 bool
 initializeJit()
    {
-   return initializeJitBuilder(0, 0, 0, (char *)"-Xjit:acceptHugeMethods,enableBasicBlockHoisting,omitFramePointer");
+   return initializeJitBuilder(0, 0, 0, (char *)"-Xjit:acceptHugeMethods,enableBasicBlockHoisting,omitFramePointer,useIlValidator");
```

Comment:
The `useIlValidator` option only makes it so that *if* validation is enabled (e.g. using `paranoidOptCheck`), the ILValidator will be used instead of the TreeVerifier. It does not enable validation by default.